app-arch/ancient/ancient-2.0.0.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Modern decompressor for old data compression formats"
+DESCRIPTION="This is a collection of decompression routines for old formats \
+popular in the Amiga, Atari computers and some other systems from 80's \
+and 90's as well as some that are currently used which were used in a some \
+specific way in these old systems."
+HOMEPAGE="https://github.com/temisu/ancient"
+COPYRIGHT="2017-2022, Teemu Suutari and contributors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/temisu/ancient/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9c6b31e4d968ec3adb25596a0b30405d1080d3de0546f1485ecb0ac2eca6261d"
+
+ARCHITECTURES="all ?x86_gcc2"
```

Comment:
Doesn't build with gcc2, so:
```diff
-ARCHITECTURES="all ?x86_gcc2"
+ARCHITECTURES="all !x86_gcc2"
```

Reply:
I will probably look at it tomorrow, and also flatten down the commits to one single.


The v1.0 recipe had this chunk, which matches the mini-guide in the wiki, which I add back in
```
ARCHITECTURES="?all !x86_gcc2"
SECONDARY_ARCHITECTURES="x86"

commandSuffix=$secondaryArchSuffix
commandBinDir=$binDir
if [ "$targetArchitecture" = x86_gcc2 ]; then
	commandSuffix=
	commandBinDir=$prefix/bin
fi
```
So even if x86_gcc is never built, the copy-paste block is a standard code block.



PROVIDES should then provide
cmd:ancient$commandSuffix = $portVersion

and in INSTALL, after `make install`, I need to do a mv if `$binDir/ancient` and `$commandBinDir/ancient$commandSuffix` does not match


or do like bzip2, do not provide commands in secondaryArch
```
	#remove bin commands and man pages when building for secondary arch
	if [ -n "$secondaryArchSuffix" ];then
		rm -rf $prefix/bin
		rm -rf $manDir
	fi
```

Reply:
> or do like bzip2, do not provide commands in secondaryArch

There is a difference if the primary binary is build, then there is no need for it in secondary architecture (as in this case for bzip2)

Reply:
Updated with !x86_gcc2 details, and flattened the commit log

Reply:
@Begasus does recipe look ok in its current shape?